backend/models/plan.py

```diff
@@ -176,11 +176,16 @@ def add_participant(self, uuid: str):
             raise PlanDBAddingError() from e
 
     def remove_participant(self, uuid: str):
-        if uuid not in self._participants:
+        if uuid not in [p.uuid if isinstance(p, User) else p for p in self._participants]:
```

Comment:
nice to point this out! If i am not wrong you could directly use self.get_plan_participants_id(). I should have done this, my bad.

Reply:
You are right, I have fixed. Thank u